src/go/rpk/pkg/cli/registry/schema/get.go

```diff
@@ -0,0 +1,129 @@
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/sr"
+)
+
+func newGetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var (
+		sversion   string
+		id         int
+		schemaFile string
+		deleted    bool
+	)
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get a schema by version, ID, or by an existing schema",
+		Long: `Get a schema by version, ID, or by an existing schema.
+
+This returns a lookup of an existing schema or schemas in one of a few
+potential (mutually exclusive) ways:
+
+* By version, returning a schema for a required subject and version
+* By ID, returning all subjects using the schema, or filtering for one subject
+* By schema, checking if the schema has been created in the subject
+`,
+		Args: cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			f := p.Formatter
+			if h, ok := f.Help([]subjectSchema{}); ok {
+				out.Exit(h)
+			}
+			p, err := p.LoadVirtualProfile(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			cl, err := schemaregistry.NewSRClient(fs, p)
+			out.MaybeDie(err, "unable to initialize schema registry client: %v", err)
+
+			var n int
+			if sversion != "" {
+				n++
+			}
+			if id != 0 {
+				n++
+			}
+			if schemaFile != "" {
+				n++
+			}
+			switch {
+			case n == 0:
+				out.Die("Must specify at least one of --version, --id, or --schema.")
+			case n == 1:
+			default:
+				out.Die("Must specify only one of --version, --id, or --schema.")
+			}
+			if len(args) == 0 && (sversion != "" || schemaFile != "") {
+				out.Die("Subject must be specified for --version or --schema.")
+			}
+
+			switch {
+			case sversion != "":
+				version, err := parseVersion(sversion)
+				out.MaybeDieErr(err)
+				s, err := cl.SchemaByVersion(cmd.Context(), args[0], version, sr.HideShowDeleted(deleted))
+				out.MaybeDieErr(err)
+				err = printSubjectSchemaTable(f, false, s)
+				out.MaybeDieErr(err)
+
+			case id != 0:
+				ss, err := cl.SchemaUsagesByID(cmd.Context(), id, sr.HideShowDeleted(deleted))
+				out.MaybeDieErr(err)
+				if len(args) == 0 {
+					err = printSubjectSchemaTable(f, false, ss...)
+					out.MaybeDieErr(err)
+					return
+				}
+				for _, s := range ss {
+					if s.Subject == args[0] {
+						err = printSubjectSchemaTable(f, false, s)
+						out.MaybeDieErr(err)
+						return
+					}
+				}
+
+			case schemaFile != "":
+				file, err := os.ReadFile(schemaFile)
+				out.MaybeDie(err, "unable to read %q: %v", err)
+				var t sr.SchemaType
+				switch {
+				case strings.HasSuffix(schemaFile, ".avro"):
+					t = sr.TypeAvro
+				case strings.HasSuffix(schemaFile, ".proto"):
+					t = sr.TypeProtobuf
+				default:
+					out.Die("unable to determine the schema type from %q", schemaFile)
+				}
+				s, err := cl.LookupSchema(cmd.Context(), args[0], sr.Schema{
+					Schema: string(file),
+					Type:   t,
+				})
+				out.MaybeDieErr(err)
+				err = printSubjectSchemaTable(f, false, s)
+				out.MaybeDieErr(err)
```

Comment:
Can these lines, printing and maybe-err-exit, be moved outside of the switch in all cases?
And then save `ss` to a variable in each case?

Basically: I want to ensure what we only call printSubjectSchemaTable once, with `f, false, ss...`, because this will help ensure if/when this function grows that we _always_ use `false` for the second argument. It helps reduce the risk that we in the future think it is ok to use `true`, which will cause a panic since the type in the help is a slice.